tutorials/monitoring/3-configuring-endpoints/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Monitoring NServiceBus solutions: Configuring endpoints"
+reviewed: 2017-10-10
+summary: Setting up NServiceBus endpoints to be monitored by the Particular Service Platform.
+extensions:
+- !!tutorial
+  nextText: "Next Lesson: Throughput and processing time"
+  nextUrl: tutorials/monitoring/4-endpoint-performance
+---
+
+include: monitoring-intro-paragraph
+
+include: monitoring-sample-solution
+
+This third lesson guides you through setting up all of the endpoints in your system to be monitored by the Particular Software Platform.
+
+
+## Configure endpoints
+
+In order to be monitored by the Particular Service Platform, each endpoint needs to be configured to send data to three different queues: error, audit, and monitoring. It is recommended to go through this entire process for a single endpoint and ensure that it is working before continuing on to the next endpoint.
+
+
+### Error
+
+Whenever an NServiceBus endpoint is unable to process a message (even after several attempts) it will forward a copy to the error queue. Each environment should contain a single error queue and should not share an error queue with another environment.
+
+The error queue is created with a new ServiceControl instance. By default, it is named `error`.
+
+You can configure the location of the error queue. Add the following code to your endpoint configuration:
+
+```csharp
```

Comment:
should this be changed to a snippet ref?

Reply:
Yeah, it will be. I was running into trouble doing that so I in-lined a few of the snippets make the process smooth for now